en-US/snippets/2016/dec2016.lang

```diff
@@ -0,0 +1,41 @@
+## NOTE: reference bug 1318151, deadline Dec 5th, 2016
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAK (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+
+
+## MAX_LENGTH: 180
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
+# Link: https://addons.mozilla.org/firefox/themes/
+;Customize. Create. Explore. <a>We’ve got themes galore</a>.
+Customize. Create. Explore. <a>We’ve got themes galore</a>.
+
+
+## MAX_LENGTH: 180
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
+# Link: https://developer.mozilla.org/docs/Learn
+;Decode the code! Learn HTML and JavaScript in our intensely straightforward <a>Learning Area</a>.
+Decode the code! Learn HTML and JavaScript in our intensely straightforward <a>Learning Area</a>.
+
+
+## MAX_LENGTH: 180
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# For PC users
+# You can use panic or worry instead of freak
+;Accidentally closed a tab? Don’t freak. There’s a quick way to get it back: Control + Shift + T. Now breathe.
+Accidentally closed a tab? Don’t freak. There’s a quick way to get it back: Control + Shift + T. Now breathe.
+
+
+## MAX_LENGTH: 180
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# For Mac users
+# You can use panic or worry instead of freak
+;Accidentally closed a tab? Don’t freak. There’s a quick way to get it back: Command + Shift + T. Now breathe.
+Accidentally closed a tab? Don’t freak. There’s a quick way to get it back: Command + Shift + T. Now breathe.
+
+
+## MAX_LENGTH: 180
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# For Mac users
```

Comment:
This is still not for Mac.

Also, can you add as suggested `# Link: none` to snippets without links?